matrixbot/matrix.py

```diff
@@ -257,6 +257,12 @@ def send_notice(self, room_id, message):
                              room_id, message)
 
     def send_private_message(self, user_id, message, room_id=None):
+        # Just add a first case: if the channel is 1-to-1 then
+        # reply directly using this channel
+        if self.is_private_room(room_id, self.get_user_id(), user_id):
+            return self.call_api("send_message", 3,
+                                 room_id, message)
```

Comment:
it is cleaner if it fits in a single line respecting the maximum length of 72

Reply:
Thanks for the suggestion. Patch updated with according with your comment.